Modules/pyexpat.c

```diff
@@ -1836,6 +1836,15 @@ add_features(PyObject *mod)
 }
 #endif
 
+static void
+pyexpat_destructor(PyObject *op)
+{
+    void *p = PyCapsule_GetPointer(op, PyExpat_CAPSULE_NAME);
+    if (p != NULL) {
+        PyMem_Free(p);
+    }
```

Comment:
The if is not needed: PyMem_Free() accepts NULL. Please remove it.

Reply:
Oh, I don't mention this detail. Updated.